src/components/Nav/Nav.scss

```
@@ -0,0 +1,205 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+.nav {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 70px;
+  color: #222222;
+  border-bottom: #eeeeee 1px solid;
+  font-weight: 700;
+
+  .nav_menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 1440px;
+
+    .header_logo {
+      flex: 1;
+      .leader_logo_image {
+        height: 33px;
+        padding-right: 35px;
+        cursor: pointer;
+      }
+    }
+
+    .header_shop_nav {
+      display: flex;
+      flex: 10;
+
+      .nav_underLine {
+        position: relative;
+        margin: 0 35px;
+        font-family: 'Roboto', sans-serif;
+        line-height: 70px;
+        cursor: pointer;
+
+        .header_sub_nav {
+          display: none;
+        }
+
+        &:hover .header_sub_nav {
+          position: absolute;
+          display: flex;
+          align-items: center;
+          height: 70px;
+          top: 100%;
+          background-color: #222222;
+          color: white;
+          z-index: 2;
+        }
+      }
+
+      li:before {
+        content: '';
+        position: absolute;
+        height: 4px;
+        width: 0;
+        bottom: 20px;
+        background-color: #fdcf41;
+        transition: 0.3s;
+      }
+
+      li:hover:before {
+        width: 100%;
+      }
+    }
+
+    .header_user_nav {
+      position: relative;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      flex: 4;
+      font-family: 'Roboto', sans-serif;
+      cursor: pointer;
+
+      .header_search,
+      .heder_alert,
+      .header_cart,
+      .header_user {
+        img {
+          width: 24px;
+          transition: 200ms cubic-bezier(0.4, 0, 0.2, 1);
+        }
+      }
+
+      .header_search,
+      .heder_alert,
+      .header_cart,
+      .header_user {
+        img:hover {
+          width: 24px;
+          filter: opacity(0.5) drop-shadow(0 0 0 #fdcf41);
+        }
+      }
+
+      .header_login,
+      .header_signUp {
+        transition: 200ms cubic-bezier(0.4, 0, 0.2, 1);
+      }
+
+      .header_login:hover,
+      .header_signUp:hover {
+        color: #fdcf41;
+      }
```

```suggestion
      .header_login,
      .header_signUp {
        transition: 200ms cubic-bezier(0.4, 0, 0.2, 1);
        &:hover {
           color: #fdcf41;
        }
      }
```
여기도 이렇게 수정할 수 있겠네요~
다른부분도 중복선언한 속성 있다면 확인 해보세요:-)

---

네엡 수정하였습니다 피드백 주신덕분에 scss 길이가 많이 줄어들었어욥 감사합니다